composer.json

```
@@ -121,6 +121,7 @@
         "symfony/filesystem": "^5.4 || ^6.0",
         "symfony/finder": "^5.4 || ^6.0",
         "symfony/framework-bundle": "^5.4 || ^6.0",
+        "symfony/html-sanitizer": "^5.4 || ^6.0",
```

```suggestion
        "symfony/html-sanitizer": "^6.1",
```

The component was first released in version `6.1`.

Can we maybe also bump the "symfony/twig-bridge" to ^6.1? Then we would also automatically get the [`HtmlSanitizerExtension`](https://github.com/symfony/symfony/blob/6.2/src/Symfony/Bridge/Twig/Extension/HtmlSanitizerExtension.php) that does the setup for the `sanitize_html` filter for free.

---

Done in 55f82e25a04a4de064ad3c328a4ce9cd9861f921.